.github/workflows/js.yml

```
@@ -0,0 +1,22 @@
+name: JS
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

You're not making a library, I think is good if this works with the latest stable version, and that's enough. What do you think?
```suggestion
          node-version: '16'
```

---

Good idea 👍 I'm setting this version (Node 16) for all three jobs.